ideas42ventures/ideas42ventures.com
# adds netlify redirect with 404 page

Works on #27 

- Added a 404 redirect.

**How to test:** 

1. The first link provided is invalid. When clicked it should redirect to a 404 page.

[Invalid URL](https://deploy-preview-31--ideas42ventures.netlify.app/2) should render a [404 page](https://deploy-preview-31--ideas42ventures.netlify.app/404.)
